schemas/csdl.schema.json

```diff
@@ -171,7 +171,7 @@
       "type": "object",
       "additionalProperties": false,
       "patternProperties": {
-        "@": {
+        "^@": {
```

Comment:
But an entity type can contain a `property@an.no.tation`.

Reply:
No, the entity type can contain annotations on the type, annotations on a property go into the object of the property definition.

Reply:
I mixed up CSDL and runtime JSON.